tests/pr_test.go

```diff
@@ -32,10 +32,10 @@ func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptio
 		TerraformDir:                  dir,
 		Prefix:                        prefix,
 		ResourceGroup:                 resourceGroup,
+		Region:                        "us-south",
```

Comment:
add a comment here:
// lock into us-south as is supports cross region support when using your own KMS key

Reply:
@jor2 comment still missing here